recipe/meta.yaml

```diff
@@ -37,6 +43,16 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    {% if cuda_major >= 12 %}
+    - cuda-driver-dev
+    - cuda-cudart-dev
+    - cuda-nvrtc-dev
+    - cuda-nvtx-dev
+    - cuda-nvml-dev
+    - libcublas-dev
+    - libcufft-dev
+    - libcurand-dev
+    {% endif %}
```

Comment:
The only ones of these we use are driver, nvrtc, and cufft.

Reply:
I just pushed https://github.com/conda-forge/openmm-torch-feedstock/pull/43/commits/2d888d1740cc6a9957f809b7bc9fc90822781b72 to see if we can trim the deps down